### DISCUSSION TITLE
STM32RTC.H "no such file or directory" on try to flash "Black F407VE"

### DISCUSSION THREAD

**Discussion by igor-rodriguez** [stm32duino/Arduino_Core_STM32#2025] [Q&A] [answered]
Arduino IDE 2.1.0,
Arduino CORE STM32 instaled,
STM32CubeProg instaled,
 IDE return this error:

In file included from C:\Users\zz1py\Downloads\SpeeduINO\speeduino-master\speeduino\globals.h:150,
                 from C:\Users\zz1py\Downloads\SpeeduINO\speeduino-master\speeduino\speeduino.ino:24:
C:\Users\zz1py\Downloads\SpeeduINO\speeduino-master\speeduino\board_stm32_official.h:7:10: fatal error: STM32RTC.h: No such file or directory
    7 | #include "STM32RTC.h"
      |          ^~~~~~~~~~~~
compilation terminated.

exit status 1

Compilation error: STM32RTC.h: No such file or directory

Someone get´s help?

**Comment by fpistm** [comment, marked as answer]
Hi @igor-rodriguez 
I' converted to a discussion as it works.
You have to install the STM32RTC library thanks the library manager:
https://github.com/stm32duino/STM32RTC
